packages/react-data-grid/style/rdg-cell.less

```
@@ -34,6 +34,14 @@
   z-index: 3
 }
 
+@supports (position: sticky) {
```

```suggestion
@supports (position: sticky) or (position: -webkit-sticky) {
```
https://developer.mozilla.org/en-US/docs/Web/CSS/@supports#The_or_operator
Browsers that only support the prefixed version won't... you know... support the unprefixed version.